### PR TITLE
New Keyboard: FLX Virgo

### PR DESCRIPTION
**Pull request by mechmerlin**
## Description

I helped Flex get his board into QMK. We have confirmed this working on his hardware. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/flx/virgo/readme.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# virgo
+
+![virgo](imgur.com image replace me!)
+
+A short description of the keyboard/project
+
+* Keyboard Maintainer: [MechMerlin](https://github.com/yourusername)
+* Hardware Supported: Virgo PCB
```
Comment (fauxpark) [inline keyboards/flx/virgo/readme.md]:
```suggestion
* Hardware Supported: Virgo PCB
```

File: keyboards/flx/virgo/readme.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# virgo
+
+![virgo](imgur.com image replace me!)
+
+A short description of the keyboard/project
+
+* Keyboard Maintainer: [MechMerlin](https://github.com/yourusername)
+* Hardware Supported: Virgo PCB  
+* Hardware Availability: GB TBD
```
Comment (fauxpark) [inline keyboards/flx/virgo/readme.md]:
```suggestion
* Hardware Availability: GB TBD
```

File: keyboards/flx/virgo/rules.mk [hunk 3]
```diff
@@ -0,0 +1,35 @@
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   atmega32a    bootloadHID
```
Comment (fauxpark) [inline keyboards/flx/virgo/rules.mk]:
```suggestion
#   ATmega32A    bootloadHID
#   ATmega328P   USBasp
```

File: keyboards/flx/virgo/virgo.c [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    writePinHigh(E6);
+  }
+
+if (IS_LED_ON(usb_led, USB_LED_SCROLL_LOCK)) {
+    writePinLow(B2);
+  } else {
+    writePinHigh(B2);
+  }
+	led_set_user(usb_led);
+}
```
Comment (fauxpark) [inline keyboards/flx/virgo/virgo.c]:
```c
bool led_update_kb(led_t led_state) {
    if(led_update_user(led_state)) {
        writePin(E6, !led_state.caps_lock);
        writePin(B2, !led_state.scroll_lock);
    }
    return true;
}
```

File: keyboards/flx/virgo/config.h [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x0000
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    [EDIT_THIS]
```
Comment (fauxpark) [inline keyboards/flx/virgo/config.h]:
```suggestion
#define MANUFACTURER    FLX
```

File: keyboards/flx/virgo/config.h [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+#define PRODUCT         [EDIT_THIS] FLX-V90 Default
```
Comment (fauxpark) [inline keyboards/flx/virgo/config.h]:
```suggestion
#define PRODUCT         Virgo
```

File: keyboards/flx/virgo/config.h [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+#define DESCRIPTION     [EDIT_THIS] A QMK-powered custom keyboard
```
Comment (fauxpark) [inline keyboards/flx/virgo/config.h]:
```suggestion
#define DESCRIPTION     A QMK-powered custom keyboard
```
... or something 😄

File: keyboards/flx/virgo/readme.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
+# virgo
+
+![virgo](imgur.com image replace me!)
+
+A short description of the keyboard/project
+
+* Keyboard Maintainer: [MechMerlin](https://github.com/yourusername)
```
Comment (fauxpark) [inline keyboards/flx/virgo/readme.md]:
```suggestion
* Keyboard Maintainer: [MechMerlin](https://github.com/mechmerlin)
```

File: keyboards/flx/virgo/config.h [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x0000
```
Comment (drashna) [inline keyboards/flx/virgo/config.h]: Product ID?

File: keyboards/flx/virgo/readme.md [hunk 10]
```diff
@@ -0,0 +1,15 @@
+# virgo
+
+![virgo](imgur.com image replace me!)
```
Comment (drashna) [inline keyboards/flx/virgo/readme.md]: Image?

**Pull request merged**